packages/test/test-service-load/src/utils.ts

```diff
@@ -174,21 +182,24 @@ export async function initialize(
 		generateConfigurations(seed, optionsOverride?.configurations),
 	);
 
-	const logger = await createLogger({
-		driverType: testDriver.type,
-		driverEndpointName: testDriver.endpointName,
-		profile: profileName,
-		runId: undefined,
-	});
-	logger.minLogLevel = random.pick([LogLevel.verbose, LogLevel.default]);
+	const minLogLevel = random.pick([LogLevel.verbose, LogLevel.default]);
+	const logger = await createLogger(
+		{
+			driverType: testDriver.type,
+			driverEndpointName: testDriver.endpointName,
+			profile: profileName,
+			runId: undefined,
+		},
+		minLogLevel,
+	);
 
 	logger.sendTelemetryEvent({
 		eventName: "RunConfigOptions",
 		details: JSON.stringify({
 			loaderOptions,
 			containerOptions,
 			configurations,
-			logLevel: logger.minLogLevel,
+			logLevel: minLogLevel,
```

Comment:
I realize this was already in this place, but is it correct? Shouldn't it be passed as a separate parameter to `sendTelemetryEvent()`?

Reply:
I am not trying to log it as verbose event. I want to log it with details just like other loader/runtime options so that in test we can know what was the log level of the logger for that container.